src/battery.c

```diff
@@ -45,12 +45,12 @@ u8 BATTERY_Check()
         PAGE_Test();
         CONFIG_SaveModelIfNeeded();
         CONFIG_SaveTxIfNeeded();
-        SPI_FlashBlockWriteEnable(0); //Disable writing to all banks of SPIFlash
+        Flash_BlockWriteEnable(0);  // Disable writing to all banks of SPIFlash
         warned |= BATTERY_CRITICAL;
```

Comment:
How about STORAGE_BlockWriteEnable instead?  these same routines potentially apply to SD_Card too (whick I know is still 'flash' but we don't call it that)